crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -478,6 +498,20 @@ pub struct ReadOnlyOwnUserIdentity {
     verified: Arc<AtomicBool>,
 }
 
+/// Specific equality check for identities that also check if there are changes
```

Comment:
Same as for the above, docs should be on the method, not on the trait. We can reuse the comment from above as well.

Reply:
Done